build.cmd

```diff
@@ -1,89 +1,57 @@
 @if not defined _echo @echo off
 setlocal enabledelayedexpansion
 
-set BatchFile=%0
-set Root=%~dp0
+@REM Show the command line arguments available if /? is provided.
+if /I "%1" == "/?" call :Usage && exit /b 1
 
-set BuildConfiguration=Debug
-set RootSuffixCmdLine=
-set OptBuild=true
-set OptDiagnostic=false
-set OptRebuild=false
-set OptDeploy=true
-set OptTest=true
-set OptIntegrationTest=false
-set OptCI=false
-set OptSign=false
-set OptIbc=false
-set OptNodeReuse=true
-set OptClearNuGetCache=false
+@REM Turn off dotnet CLI logo
+set DOTNET_NOLOGO=true
 
-:ParseArguments
-if    "%1" == ""                                                                                    goto :DoneParsing
-if /I "%1" == "/?"                                                                                  call :Usage && exit /b 1
-if /I "%1" == "/build"                set "OptBuild=true"  && set "OptRebuild=false"                && shift && goto :ParseArguments
-if /I "%1" == "/no-build"             set "OptBuild=false" && set "OptRebuild=false"                && shift && goto :ParseArguments
-if /I "%1" == "/rebuild"              set "OptBuild=false" && set "OptRebuild=true"                 && shift && goto :ParseArguments
-if /I "%1" == "/test"                 set "OptTest=true"                                            && shift && goto :ParseArguments
-if /I "%1" == "/no-test"              set "OptTest=false"                                           && shift && goto :ParseArguments
-if /I "%1" == "/integration"          set "OptIntegrationTest=true"                                 && shift && goto :ParseArguments
-if /I "%1" == "/no-integration"       set "OptIntegrationTest=false"                                && shift && goto :ParseArguments
-if /I "%1" == "/deploy"               set "OptDeploy=true"                                          && shift && goto :ParseArguments
```

Comment:
I do not believe this worked after the roslyn-tools removal. For now, `/p:DeployExtension=false` will do this. That is noted in the README.md for those using Build Tools for VS.